### ISSUE TITLE
coinbase reward vs. tx fees (denominations correct?)

### ISSUE THREAD

**Issue by antiochp** [mimblewimble/grin#255]
Is the tx fee correct in relation to the block reward?
(Also please double check my understanding of the values below, I may be off...)

```
Block reward:      50.0
Tx fee:             0.00000008
Tx fee (1000 txs):  0.00008
```

Tx fees may also be smaller than this as the wallet impl aims to reduce the UTXO set on every tx (maximizes the number of outputs being spent).

You'd need to successfully mine more than 10,000 blocks, each containing 1,000 txs to earn 1.0 grin from the tx fees?

If the tx fees are purely an anti-spam measure then it makes sense, but if the fees are to incentivize miners to select specific txs - is this high enough?

**Comment by antiochp** [comment]
From @ignopeverell on gitter -
> regarding base fee, we may be off by an order of magnitude at least to start with, I wanted 0.001% of the reward

**Comment by ignopeverell** [comment]
Fixed by 6ac2fe2.
